src/server/report/test-result.js

```diff
@@ -37,9 +39,10 @@ class TestResult extends LitElement {
 			position: absolute;
 			top: 0;
 			left: 0;
+			z-index: 1;
```

Comment:
This was required so that the sticky settings bar can float overtop of the diff overlay.